src/pages/AddItem.js

```diff
@@ -25,7 +25,7 @@ const AddItem = () => {
     if (shoppingList.length === 0) {
       fetchList(token);
     }
-  }, [fetchList, shoppingList, token]);
+  }, []); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
What did you run into that led you to disable this ESLint check here?

Reply:
i honestly don't know how ESLint was disabled?

Reply:
@segdeha eslint is disabled here so that we can pass an empty array for deps. We only want this `useEffect` to execute after the first render.